client/src/pages/guide/english/ssh/index.md

```diff
@@ -8,3 +8,20 @@ Secure Shell (SSH) is a cryptographic network protocol intended for secure data
 It is most commonly used for accessing remote servers.
 
 SSH is also used in distributed source code versioning software like git.
+
+SSH works on port 22 and also used by scp to send files remotely.
+
+### Installation
+
+On Ubuntu:
+```
+sudo apt install openssh-server
+````
+On CentOS:
+```
+sudo yum install -y openssh-server
+```
+On Arch Linux:
+```
+sudo pacman -S openssh
+```
```

Comment:
I'm not sold on the value of this. It mentions just a tiny subset of platforms ssh runs on, but more importantly, just installing it does nothing. One needs to configure the server and run it.